esp-radio/README.md

```diff
@@ -24,6 +28,8 @@ If a cell contains an em dash (&mdash;) this means that the particular feature i
 | ESP32-S2 |                          ✓                           |                       &mdash;                       |                       &mdash;                        |    ✓    |
 | ESP32-S3 |                          ✓                           |                          ✓                          |                          ✓                           |    ✓    |
 
+`IEEE 802.15.4` is only available on ESP32-C6 and ESP32-H2
```

Comment:
This won't be the case forever. You can add another column I guess, and also change `Coex` to `Coex (Wi-Fi + BLE)`.